RemoteInput/Plugin/ControlCenter.hxx

```diff
@@ -27,6 +26,26 @@ enum class ReflectionArrayType: std::uint8_t
 	OBJECT
 };
 
+enum class ReflectionBinaryType : std::uint8_t
```

Comment:
No need for this. It can inherit `ReflectionArrayType` or just rename that type and add `ARRAY` to it instead of having two enum types with almost the exact same values + 1.